ARC specification.md

```diff
@@ -446,10 +564,48 @@ validation_packages:
   - name: package3
 ```
 
+### ARC Apps
+
+Continuous Quality Control enables to check at any time in the ARC life cycle whether it passes certain criteria or not.
+
+However, **if** an ARC is valid for a given _target_ is only half of the equation - the other being taking some kind of action based on this information. One large field of actions here is the publication of the ARC or (some) of it's contents to an **endpoint repository (ER)** (e.g. [PRIDE](https://www.ebi.ac.uk/pride/), [ENA](https://www.ebi.ac.uk/ena/browser/home)).
+
+In this example, a validation package SHOULD only determine if the content _COULD_ be published to the ER, and a subsequent service SHOULD then take the respective action based on the reported result of that package (e.g. fixing errors based on the report, or publish the content to the ER).
+
+**ARC apps** are services that provide URLs called _(CQC) Hook Endpoints_ that be triggered manually or by the result of a validation package. They are intended to automate the process of taking action based on the result of a validation package.
+
 ### Reference implementation
 
 PLANTDataHUB performs Continuous Quality Control of ARCs using the [arc-validate software suite](https://github.com/nfdi4plants/arc-validate) as described in our 2023 paper [PLANTdataHUB: a collaborative platform for continuous FAIR data sharing in plant research](https://doi.org/10.1111/tpj.16474).
 
+The following sequence diagram shows the conceptual implementation of CQC pipelines in conjunction with ARC Apps connected via CQC Hooks on the reference DataHUB instance with the following participants:
+
+- **User**: The user who works on an ARC published on the DataHUB
+- **ARC**: The ARC repository on the DataHUB
+- **DataHUB**: The DataHUB instance
+- **ARC App**: A service that provides a CQC Hook Endpoint to perform actions based on validation results and/or user input
+
+```mermaid
+sequenceDiagram
+
+    participant User
+    participant ARC
+    participant DataHUB
+    participant ARC App
+
+    Note over User, DataHUB: Validation (CQC pipeline)
+    User ->> ARC : commit
+    DataHUB ->> DataHUB : trigger validation for commit
+    DataHUB ->> ARC : commit validation results <br> to cqc branch
+    DataHUB ->> ARC : create badge
+    Note over User, ARC App: QCQ Hooks
```

Comment:
cqc?

Reply:
If this question should imply what cqc is, that is explained in the very same paragraph

Reply:
Ah no just wanted to mention that you wrote qcq in this line

Reply:
lol i was completely blind to this